io.greenfire.Greenery.json

```diff
@@ -0,0 +1,87 @@
+{
+    "app-id": "io.greenfire.Greenery",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "23.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "greenery",
+    "separate-locales": false,
+    "tags": [
+        "proprietary"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            "name": "greenery",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 io.greenfire.Greenery.desktop /app/share/applications/io.greenfire.Greenery.desktop",
+                "install -Dm644 io.greenfire.Greenery.appdata.xml /app/share/appdata/io.greenfire.Greenery.appdata.xml",
+                "install -Dm644 -t /app/share/icons/hicolor/scalable/apps io.greenfire.Greenery.svg",
+                "install -Dm644 io.greenfire.Greenery-128x128.png /app/share/icons/hicolor/128x128/apps/io.greenfire.Greenery.png",
+                "install -Dm644 io.greenfire.Greenery-64x64.png /app/share/icons/hicolor/64x64/apps/io.greenfire.Greenery.png",
```

Comment:
```suggestion
```

that too

Reply:
I have also removed these lines from build-commands